docs/bokeh/source/docs/user_guide/interaction/tooltips.rst

```diff
@@ -71,7 +71,7 @@ Currently, the following input widgets support tooltips directly:
 
 * :ref:`ug_interaction_widgets_examples_autocompleteinput`
 * :ref:`ug_interaction_widgets_examples_colorpicker`
-* :ref:`ug_interaction_widgets_examples_datepicker`
+* :ref:`ug_interaction_widgets_examples_date_picker`
```

Comment:
I would expect the original to fail compilation. I wonder how many more refs are broken across the docs.

Reply:
It failed for me locally, which is how I found it. Not sure why it didn't in CI